languages/typescript/src/test/resources/typescript-code/tslint-vscode.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Is this file automatically used by vscode? And why do we have to use a separate tslint file for vscode?

Reply:
Its actually just a copy paste from the node repo that i use to test the quality of my generated code, but as long as we don't have a correct ci, i just have to use it mannually wenever i do work on the node sdk 😢